tests/external_jwt_signer_test.go

```diff
@@ -518,12 +605,13 @@ func Test_ExternalJWTSigner(t *testing.T) {
 			jwtSignerEnabled := true
 
 			jwtSigner := &rest_model.ExternalJWTSignerCreate{
-				CertPem:  &jwtSignerCertPem,
-				Enabled:  &jwtSignerEnabled,
-				Name:     &jwtSignerName,
-				Kid:      S(uuid.New().String()),
-				Issuer:   S(uuid.NewString()),
-				Audience: S(uuid.NewString()),
+				CertPem:     &jwtSignerCertPem,
+				Enabled:     &jwtSignerEnabled,
+				Name:        &jwtSignerName,
+				Kid:         S(uuid.New().String()),
+				Issuer:      S(uuid.NewString()),
+				Audience:    S(uuid.NewString()),
+				TargetToken: ToPtr(rest_model.TargetTokenID),
```

Comment:
should there be a test when it's left empty or when it's an invalid value? i didn't see such a test but it's kinda hard to scan for a missing value